python/hsfs/connection.py

```diff
@@ -184,6 +184,8 @@ def connect(self):
                 self._engine is None and not importlib.util.find_spec("pyspark")
             ):
                 self._engine = "hive"
+            elif self._engine is not None and self._engine.lower() == "hops_experiment":
```

Comment:
I wouldn't necessarily call it "hops-experiment", but I also have no strong opinion what else it could be. so fine with me.

Reply:
may be `tensorflow` or `tf_data`?